cmd/exo/run.go

```diff
@@ -68,8 +68,8 @@ If a workspace does not exist, one will be created in the current directory.
 		(func() {
 			ctx, stop := signal.NotifyContext(ctx, os.Interrupt, syscall.SIGTERM)
 			defer stop()
-			var logRefs []string
-			if err := tailLogs(ctx, workspace, logRefs); err != nil {
+			refs := []string{}
```

Comment:
This was intentionally nil before, to signal "no logs specified" differently than "specified no logs". I think you can leave this point alone and change the `len(logRefs) != 0` to `logRefs != nil`

Reply:
Yeah this is leftover from my first attempt at fixing. Will revert.